crt_portal/static/js/ga_util.js

```diff
@@ -7,25 +7,55 @@ function gtag() {
   dataLayer.push(arguments);
 }
 
-export function sendGAPublicClickEvent(event_name) {
+function sendGAPublicClickEvent(event_name) {
   gtag('event', 'click', { event_name: event_name });
 }
 
-export function sendGAClickEvent(event_name) {
+function sendGAClickEvent(event_name) {
   const section = getSection();
   gtag('event', 'click', { event_name: event_name, section: section });
 }
 
-export function sendGAFilterEvent(params) {
-  const section = getSection();
-  gtag('event', 'search_filter', { filters: params, section: section });
-}
-
 (function(root, dom) {
   function init() {
-    const relatedReportButton = dom.getElementById('related-report');
+    const relatedReportButton = dom.getElementById('related-reports');
     if (relatedReportButton !== null) {
-      relatedReportButton.addEventListener('click', sendGAClickEvent('view related reports'));
+      relatedReportButton.addEventListener('click', e => {
+        sendGAClickEvent('view related reports');
+      });
+    }
+    const navItems = document.getElementsByClassName('usa-nav__primary-item');
+    for (var i = 0; i < navItems.length; i++) {
+      const navItem = navItems[i];
+      navItem.addEventListener('click', e =>
+        sendGAPublicClickEvent('main nav ' + e.target.innerText)
+      );
+    }
```

Comment:
I don't think you actually need the index, here:

```suggestion
    navItems.forEach(navItem => {
      navItem.addEventListener('click', e =>
        sendGAPublicClickEvent('main nav ' + e.target.innerText)
      );
    });
```